src/graph.rs

```diff
@@ -16,14 +17,7 @@ use std::marker::PhantomData;
 /// Used to collect graph data during a load test.
 pub(crate) struct GraphData {
     /// Tracks when the load test first started with an optional system timestamp.
-    starting: Option<DateTime<Utc>>,
-    /// Tracks when all [`GooseUser`](../goose/struct.GooseUser.html) threads fully
-    /// started with an optional system timestamp.
     started: Option<DateTime<Utc>>,
```

Comment:
Why is this needed? Is this missing in `metrics.history`?

Reply:
Removed